src/payment/checkout/Checkout.jsx

```diff
@@ -156,12 +156,34 @@ class Checkout extends React.Component {
 
     // Stripe element config
     // TODO: Move these to a better home
-    const appearance = {
-      theme: 'stripe',
-    };
     const options = {
       clientSecret: this.props.clientSecretId,
-      appearance,
+      appearance: {
+        // Normally these styling values would come from Paragon,
+        // however since stripe requires styling to be passed
+        // in through the appearance object they are currently placed here.
+        // TODO: Investigate if these values can be pulled into javascript from the Paragon css files
```

Comment:
Oh! When previously discussing this, I hadn't understood we were hardcoding [Paragon colors](https://paragon-openedx.netlify.app/foundations/colors). I thought we were hardcoding [our company's brand colors](https://github.com/edx/brand-edx.org/blob/master/paragon/_variables.scss). In this case, they happen to be the same, so I'm not as concerned about hardcoding them.

I looked into [`getComputedStyle()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle), but it looks complicated to use in React pre-render.

@wdrussell2015, could you create a ticket for this `TODO` line with any thoughts you have for future work?

Reply:
https://2u-internal.atlassian.net/browse/REV-3212